jubeatripples.html

```diff
@@ -11,18 +11,14 @@
         <script type="text/javascript">
             window.addEventListener("load", function() {
                 new PatchContainer([
-                    new Patcher("jubeat.exe", "2009-12-24 (China ver.)", [
+                    new Patcher("jubeat.exe", "2009-12-24 (China Ver.1)", [
```

Comment:
Don't version patches like this - it's still for China ver

Reply:
I added it because there are three Chinese versions. Should I really rollback it?

Reply:
thanks! I'm done with all the modifications